src/flexmock/_api.py

```diff
@@ -76,6 +76,10 @@ def __exit__(
     ) -> None:
         pass
 
+    def __call__(self, *args: Any, **kwargs: Any) -> "Mock":
+        """Make mocks callable with and without parentheses."""
```

Comment:
Let's put the rationale in here also so se don't delete it again later 😁

Reply:
I updated the docstring:
```python
    def __call__(self, *args: Any, **kwargs: Any) -> "Mock":
        """Make mocks callable with and without parentheses.

        If `Mock` is not callable, it is difficult to mock attributes that
        should work with and without parentheses.
        """
        return self
```